url-shortner.js

```diff
@@ -0,0 +1,52 @@
+/*
+1. http://localhost:3000/ -> we get a form
+2. we submit the form
+3. app generates an random id like 234234 => function storeShortUrl
+4. return the the front page where we see the new short url in the table below
+5. when we visit http://localhost:3000/234234 it redirects to the full url.
+*/
```

Comment:
nit: Might be better to put these types of comments in the README.md file, or a NOTES.md file, or remove it from the repository altogether.